File: BASE/PythonBase/정규표현식/원하는부분추출.py
import re
text = "From chlgmltn101@naver.com Sat Jan  5 09:14:16 2019"
# 1. \S 공백 아닌 문자 찾기
string1 = re.findall('\S+@\S+', text)
# 2. 소괄호
string2 = re.findall('From (\S+@\S+)', text)

# 이메일 호스트 출력
string3 = re.findall('@([^ ]+)', text)
print(string1)
print(string2)
print(string3)

File: 자스베이스/정규표현식/기본/원하는부분추출.js
const text = "From chlgmltn101@naver.com Sat Jan  5 09:14:16 2019";
const string1 = /\S+@\S+/g.exec(text);
// 2. 소괄호
const string2 = /From (\S+@\S+)/g.exec(text);
// 이메일 호스트 출력
const string3 = /@([^ ]+)/g.exec(text);
console.log(string1[0]);
console.log(string2[0]);
console.log(string3[0]);
